### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by koansang**
- separate the module and endpoint by API
- Add `getApis()` in client.js 
- Add description for api call in readme
- Add a previously created function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,20 @@ const api = new EnerTalkAPI(authConfig, {
...
+There are two ways to get an Api object. It's `getApi(key)` and `getAllApis()`.   
+`key` is name of API.
+
+For example,
+
+```js
+//enerTalkApi is instance of EnerTalkAPI
+
+const billApi = enerTalkApi.getApi('bill');
+const allApis = enerTalkApi.getAllApis();
```
Comment (yongdamsh) [inline README.md]: For a consistent convention, how about use `API` instead of `Api`?

File: lib/__tests__/client.test.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const APIClient = require('../client');
 const constants = require('../constants');
+const { bills } = require('../apis');
```
Comment (yongdamsh) [inline lib/__tests__/client.test.js]: I think we should support node v4. It is necessary for services like *AWS Lambda*.
The 'destructuring assignment' doesn't supported.

Please refer to [this commit](https://github.com/yongdamsh/enertalk-api-client/commit/17c4dc4170ff68a8f625403df32a781209a7d51c).

File: lib/client.js [hunk 3]
```diff
@@ -146,77 +147,20 @@ class EnerTalkAPIClient extends axios.Axios {
     });
   }
 
-  getUser() {
```
Comment (yongdamsh) [inline lib/client.js]: How about keeping these methods for backward compatibility?

File: README.md [hunk 4]
```diff
@@ -82,6 +82,20 @@ const api = new EnerTalkAPI(authConfig, {
...
+## Get API
+There are two ways to get an Api object. It's `getApi(key)` and `getAllApis()`.   
+`key` is name of API.
+
+For example,
+
+```js
+//enerTalkApi is instance of EnerTalkAPI
+
+const billApi = enerTalkApi.getApi('bill');
```
Comment (koansang) [inline README.md]: ok~ I will update soon.^^

File: README.md (hunk 1 above)
Comment (koansang) [inline README.md]: you are right~!!

File: lib/__tests__/client.test.js (hunk 2 above)
Comment (koansang) [inline lib/__tests__/client.test.js]: I see. How about deploying files build using Webpack and Babel?

Comment (yongdamsh) [inline lib/__tests__/client.test.js]: Good idea, I'll apply it soon.

File: lib/apis/push/index.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  constructor(axios) {
+    this.axios = axios;
+  }
+
+  registerPushId(data = {}) {
+    return this.axios.post(endpoint.pushInfo, data);
+  }
+
+  getPushInfo() {
+    return this.axios.get(endpoint.pushInfo);
```
Comment (yongdamsh) [inline lib/apis/push/index.js]: The `mobileId`(UUID format) parameter seems to be missing.

Comment (koansang) [inline lib/apis/push/index.js]: ok~thanks~^^

File: lib/client.js (hunk 3 above)
Comment (koansang) [inline lib/client.js]: ok~

**Pull request merged**